cartoframes/context.py

```diff
@@ -1643,6 +1647,11 @@ def data(self, table_name, metadata, persist_as=None, how='the_geom'):
         return self.query(query,
                           table_name=persist_as)
 
+    def _unique(self, suggested, existing):
```

Comment:
This function has a more natural home in [`cartoframes.utils.py`](https://github.com/CartoDB/cartoframes/blob/master/cartoframes/utils.py) with the other column naming functions since it doesn't need access to `self`.